shell/platform/android/io/flutter/view/ResourceExtractor.java

```diff
@@ -202,17 +203,23 @@ private String checkTimestamp(File dataDir) {
         final String[] existingTimestamps = getExistingTimestamps(dataDir);
 
         if (existingTimestamps == null) {
-            Log.i(TAG, "No extracted resources found");
+            if (BuildConfig.DEBUG) {
+                Log.i(TAG, "No extracted resources found");
```

Comment:
@sbaranov perhaps this should be `Log.e` or `Log.w` instead?

Reply:
It's not an abnormal condition, so Log.i is fine.